### PR TITLE
List Serve functionality

### PR DESCRIPTION
**Pull request by JacobRoberge**
tested on heroku
;)

### CODE REVIEW

File: server/controllers/mail.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+//const mailjet = require("node-mailjet").connect(
+//    process.env.MAILJET_KEY_PUBLIC,
+//    process.env.MAILJET_KEY_PRIVATE
+//);
+
 const mailjet = require("node-mailjet").connect(
-    process.env.MAILJET_KEY_PUBLIC,
-    process.env.MAILJET_KEY_PRIVATE
+    '6985f34c9cd9618795ddd13715d53d30',
+    '3dc0cff44753d80c8592e6073f660be1'
```
Comment (mohona17) [inline server/controllers/mail.js]: I dont think these keys should be public

File: server/models/emailList.server.model.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    var currentDate = new Date();
+    this.updated_at = currentDate;
+    if (!this.created_at) {
+        this.created_at = currentDate;
+    }
+    next();
+});
+
+var emailList = mongoose.model('Listing', listingSchema);
+module.exports = emailList;
```
Comment (mohona17) [inline server/models/emailList.server.model.js]: Can you clean this up so it says email instead of listing

File: client/src/views/Home/Home.js [hunk 3]
```diff
@@ -74,18 +91,42 @@ function Home() {
...
+                        listServe('testy', 'wazoo is an odd word').then(({ message }) => {
+                            alert(message);
+                        });
+                    }}
+                >
+                    list serve test
+                    </button>
+            </div>
+        );
+    }
```
Comment (mohona17) [inline client/src/views/Home/Home.js]: Why are there two buttons? Can you clean this up? Can you change the alert?

File: .vs/VSWorkspaceState.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "\\client\\src\\components\\Header",
+    "\\client\\src\\views",
+    "\\client\\src\\views\\Home",
+    "\\server",
+    "\\server\\models",
+    "\\server\\routes"
+  ],
+  "SelectedNode": "\\server\\controllers\\mail.js",
+  "PreviewInSolutionExplorer": false
+}
```
Comment (mohona17) [inline .vs/VSWorkspaceState.json]: What is this file for? (I am just curious)

Comment (JacobRoberge) [reply .vs/VSWorkspaceState.json]: I think visual studio added it automatically.

**Comment by mohona17** [comment]
Branch is not updated with master.

**Pull request closed without merging**